socorro/external/postgresql/staticdata.py

```diff
@@ -1,3 +1,5 @@
+# flake8: noqa
```

Comment:
This file has a bunch of long lines. It's just data. We could move the data to CSV files, but seemed like a better trade-off to just noqa the whole file.